resource/systemd/states.go

```diff
@@ -66,13 +66,13 @@ const (
 	ASDeactivating ActiveState = "deactivating"
 )
 
-func (state ActiveState) Equal(state2 ActiveState) bool {
-	if state == state2 {
+func (state ActiveState) Equal(other ActiveState) bool {
+	if state == other {
 		return true
 	}
-	state = ActiveState(strings.Replace(string(state), "\"", "", -1))
-	state2 = ActiveState(strings.Replace(string(state2), "\"", "", -1))
-	return state == state2
+	stateStr := strings.Trim(string(state), "\"")
+	otherStr := strings.Trim(string(other), "\"")
+	return stateStr == otherStr
```

Comment:
Occasionally property values are "\"<property-value>\"". sometimes they are not. I have to take both into account.